database/boltdb.go

```diff
@@ -133,3 +123,23 @@ func (db boltDriver) ListNonTerminalTasks() ([]*types.EremeticTask, error) {
 
 	return tasks, err
 }
+
+func (db boltDriver) Count() int {
+	var tasks []*types.EremeticTask
+
+	db.database.View(func(tx *bolt.Tx) error {
```

Comment:
doing a full scan of the bucket to get the size feels very dangerous. Could we use a [sequence](https://github.com/boltdb/bolt#autoincrementing-integer-for-the-bucket) instead